test/edu/ithaca/dragonlab/ckc/conceptgraph/ConceptGraphTest.java

```diff
@@ -236,6 +238,17 @@ public void calcKnowledgeEstimateMoreComplexTest() {
         Assert.assertEquals(0.85, orig.findNodeById("Counting").getKnowledgeEstimate(), DataUtil.OK_FLOAT_MARGIN);
         Assert.assertEquals(0.7666, orig.findNodeById("For Loop").getKnowledgeEstimate(), DataUtil.OK_FLOAT_MARGIN);
 
+
+
+
+    }
+
+    @Test
+    public void ancestryTest(){
```

Comment:
This doesn't look like a test, and since ancestry is no longer the function name, this should be deleted, and there should exist tests for the new function